assets/css/tablesorter.theme.glotpress.css

```diff
@@ -19,6 +19,7 @@ GlotPress Theme
 	cursor: pointer;
 	text-align: center;
 	border: 1px solid #eee;
+	padding-right: 20px !important;
```

Comment:
Sorry, but I'm not going to approve any new additions of `!important` rules.`!important` is definitely not "very good CSS".

> We first set the padding correctly with the tablesorter css, then override it with the table css and then set it back to what it should have been in the first place with yet another selector.

This just means that something is wrong with our table CSS specificity.